src/b/component.ts

```diff
@@ -10,6 +10,9 @@ export class Profile {
     // you'll find a bug! Have a go at fixing it and make sure your
     // tests pass. I think we can use the user isAdmin field :)
     public get isAdmin(): boolean {
-        return false;
+        if (!this.user) {
+            throw new Error('User not set');
+        }
+        return this.user.isAdmin;
```

Comment:
this isn't an unreasonable solution, another option is to return false when the user is undefined and we don't need to handle a thrown error which can sometimes be a tricky on the front-end.
```suggestion
        return this.user?.isAdmin ?? false;
```